compiler-core/templates/gleam@@compile.erl

```diff
@@ -5,7 +5,18 @@
 
 -record(arguments, {lib = "./", out = "./", modules = []}).
 
-main(Args) ->
+main(_) -> main().
+
+main() ->
+    case io:get_line("") of
+        eof -> ok;
+        Line ->
+            Args = string:split(string:trim(Line), [31], all), % Unit Separator
+            erlang:display(run(Args)),
```

Comment:
Debugging left in here?

Reply:
No; this is the output the build tool waits for to know that the erlang process has finished.
I've made it more explicit by using `io:put_chars` instead.